### PR TITLE
Merge branch 'master' of github.com:RobotLocomotion/drake into kuka-dev

### PR DESCRIPTION
**Pull request by YeZhao**
- Manually make changes in kuka torque control to comply with new rigid body tree. 

- Fix kuka simulation seg faults

- New kuka simulation becomes faster now.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -16,9 +16,6 @@
 [submodule "externals/spotless"]
 	path = externals/spotless
 	url = https://github.com/RobotLocomotion/spotless-pod.git
-[submodule "externals/director"]
-	path = externals/director
-	url = https://github.com/RobotLocomotion/director.git
```
Comment (kuindersma) [inline .gitmodules:21]: I assume this is something that happened in drake master?

**Review by kuindersma — COMMENTED** [review]
There are way too many changes to check, but I assume you've confirmed that sim+hardware works as before?

File: .gitmodules (hunk 1 above)
Comment (YeZhao) [reply .gitmodules:21]: Sorry, my mistake. This submodule should be there. I've added it back to .gitmodules. (Originally, I was manually git pull the RobotLocomotion/director with the same SHA used by RobotLocomotion/drake master).

**Comment by YeZhao** [comment]
For the IK planning, simulation and hardware work as before. I tested them already. 
For the dircol planning, there is still a reference issue (due to the change of RigidBodyTree instantiation). I'll talk with Patrick to fix it. It seems not a big problem.

**Comment by YeZhao** [comment]
Let me git pull this latest version and build from scratch for a double check.

**Pull request merged**